extension/partiql-extension-ion/resources/test/test.ion

```diff
@@ -0,0 +1,220 @@
+///////
+// These values taken from https://amazon-ion.github.io/ion-docs/docs/spec.html
```

Comment:
Just curious which values were omitted from https://amazon-ion.github.io/ion-docs/docs/spec.html? You had mentioned that the tests marked w/ `ERROR` were omitted but seems like some others weren't included such as 

```
'' :: 1                                  // An empty annotation
```

Reply:
In the future we could also take examples from https://github.com/amazon-ion/ion-tests/tree/main/iontestdata/good

Reply:
I think the middle two lines from below were left out unintentionally. Note the `jpeg` line is invalid, as the actual blob data is elided, and the final line is an error.

```
jpeg :: {{ ... }}                        // Indicates the blob contains jpeg data
bool :: null.int                         // A very misleading annotation on the integer null
'' :: 1                                  // An empty annotation
null.symbol :: 1                         // ERROR: type annotation cannot be null 
```

Reply:
I added a test that iterates over the files in https://github.com/amazon-ion/ion-tests/tree/main/iontestdata/good